calicoctl/commands/clientmgr/client.go

```diff
@@ -45,7 +47,11 @@ func NewClient(cf string) (*client.Client, error) {
 // otherwise will load from environment variables.
 func LoadClientConfig(cf string) (*api.CalicoAPIConfig, error) {
 	if _, err := os.Stat(cf); err != nil {
-		log.Infof("Config file cannot be read - reading config from environment")
+		if (cf != constants.DefaultConfigPath) {
```

Comment:
You're right, in general, though there is the following exception:
https://github.com/bcreane/calicoctl/blob/master/calicoctl/calicoctl.go#L94

But since 99% of the error messages are going to stdout, I'll respin the PR again and do the same.